lab02/sections/results.tex

```diff
@@ -30,21 +32,37 @@ \section{Results}\label{sec:results}
 			12.0 & 0.136 & -17.303 & -33   \\
 		\bottomrule
 	\end{tabular}
-	\caption{Response of Fig. \ref{fig:schematic} to frequency change ($V_x = \SI{1}{\volt}$)}
+	\caption{Response of Fig.~\ref{fig:schematic} to frequency change ($V_x = \SI{1}{\volt}$)}
 	\label{table:data}
 \end{table}
 
+\subsection{Analysis}
+
+Table~\ref{table:data} shows the response of the phase lag circuit to a range of frequencies, starting at 500\si{\hertz} then 1\si{\kilo\hertz} steps from 1\si{\kilo\hertz} to 12\si{\kilo\hertz}.
+Fig.~\ref{fig:response} shows a visual representation of the magnitude and phase shift of the output over the range of 500\si{\hertz} to 12\si{\kilo\hertz}.
+The measured frequency response shown in Fig.~\ref{fig:response} closely resembles the predicted, simulated response shown in Fig.~\ref{fig:response-sim}.
+
+\begin{figure}[tbph]
+	\centering
+	\includegraphics[width=0.8\linewidth]{graphics/response}
```

Comment:
Size seems a little small and the scale might be difficult to read when printed. Can you make the font on the axes and caption similar size?